.github/workflows/workflow.yml

```diff
@@ -3,30 +3,76 @@ on:
   # Ensure GitHub actions are not run twice for same commits
   push:
     branches: [main]
-    tags: ['*']
   pull_request:
     types: [opened, synchronize, reopened]
 jobs:
   build:
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 30
+    timeout-minutes: 10
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        node-version: ['*']
+        node-version: ['lts/*']
```

Comment:
`lts/*` if the official way to use LTS version https://github.com/actions/setup-node/tree/9ced9a43a244f3ac94f13bfd896db8c8f30da67a#supported-version-syntax

Reply:
Don't we want to test on the latest version instead of the latest LTS?
For example, right now, we're testing on Node 17, but the latest LTS is Node 16 instead. Some of our users might use Node 17, so it's helpful to test for them too.

Reply:
`*` seems to be using LTS too https://github.com/netlify/build/runs/5424064781?check_suite_focus=true#step:4:9

Reply:
Not good. We should fix.

Reply:
Apparently, only specific versions, `lts/*` and `*` are supported.

https://github.com/actions/setup-node#supported-version-syntax

https://github.com/actions/setup-node/blob/9ced9a43a244f3ac94f13bfd896db8c8f30da67a/src/installer.ts#L221

There was a PR which was opened at some point to allow more version ranges (using one of my libraries :) ) but it was closed: https://github.com/actions/setup-node/pull/239

So we have two choices:
  1. Keeping it as is, i.e. we might be missing some bugs that appear only on the last major release number. That's rare, but I think it actually did happen to us once.
  2. Using `17` instead of `*` in all our repositories. This would need to be updated every 6 months, i.e. would need some maintenance work.

Based on this, I am leaning towards `1.`, but what do you think?

Reply:
Yes, `1.` makes sense to me.

Reply:
On the plus side, 6 months out of 12, the LTS happens to coincide with the latest version, so this is only a problem half of the year :D